src/devices/cpu/axc51/axc51.cpp

```diff
@@ -406,53 +416,56 @@ void axc51base_cpu_device::do_sub_flags(uint8_t a, uint8_t data, uint8_t c)
 	SET_OV((result1 < -128 || result1 > 127));
 }
 
-void axc51base_cpu_device::transmit_receive(int source)
+uint32_t axc51base_cpu_device::get_dptr0_with_autoinc(uint8_t auto_inc)
 {
-}
-
-void axc51base_cpu_device::update_timer_t0(int cycles)
-{
-}
-
-
-
-void axc51base_cpu_device::update_timer_t1(int cycles)
-{
-}
-
-void axc51base_cpu_device::update_timer_t2(int cycles)
-{
-}
-
-void axc51base_cpu_device::update_timers(int cycles)
-{
-	while (cycles--)
+	uint32_t addr = external_ram_iaddr(DPTR0, 0xFFFF);
```

Comment:
Use lowercase hex if you want to be consistent with yourself.

Reply:
yeah, there were a few other cases of capitals already in the code on which this was based, cleaned those up too